src/app/mechanic/home/OrderDetails.js

```diff
@@ -28,6 +33,18 @@ const OrderDetails = ({ order }) => {
     setTimeout(() => setIsCopied(false), 5000);
   };
 
+  const submitPart = useCallback((event) => {
```

Comment:
Probably not the way it should be done

Reply:
użyj useForm() tak jak w AddOrder wykorzystuąc handleSubmit w formie

Reply:
nie ma czasu/nie umiem